Sources/SwiftCommand/ChildProcess.swift

```diff
@@ -5,6 +5,18 @@ import Foundation
 import WinSDK
 #endif
 
+#if os(macOS) || os(iOS)
+import Darwin
+#elseif canImport(Glibc)
+import Glibc
+#elseif canImport(Musl)
+import Musl
+#elseif os(Windows)
+import ucrt
+#else
```

Comment:
```suggestion
#elseif !os(Windows)
```

`ucrt` isn't used at all in this file.